themes/navy/source/css/_partial/sidebar.styl

```diff
@@ -8,8 +8,11 @@
   @media mq-normal
     display: block
 
+  .inner
+    margin-top: 40px
+    margin-bottom: 40px
+
 .sidebar-title
-  margin-top: 40px
```

Comment:
Remove `margin-top` as toc-inner & sidebar-inner already have `margin-(top|bottom)`

Reply:
![image](https://user-images.githubusercontent.com/15902347/84990550-dcb07d80-b177-11ea-8278-fbbfc1b183a2.png)

Is this the expected change?

Reply:
@jiangtj Fixed.